web/README.md

```diff
@@ -0,0 +1,64 @@
+# Web Development
+
+To keep any form of web development consistent please adhere to these guidelines:
+- Use [TypeScript](https://www.typescriptlang.org/) - this will promote better maintainability, early bug catching
+- Follow common [code style](#Code-Style) 
+- Use code linter tools: [ESLint](https://eslint.org/docs/about/), [Prettier](https://prettier.io/docs/en/index.html)
+- Preferred package manager is [Yarn](https://yarnpkg.com/) 
+
+## Code Style
+
+Here are configuration snippets.
+
+### Prettier configuration snippet
+
+```json
+{
+  "semi": false,
+  "trailingComma": "all",
```

Comment:
Not sure about trailing commas, not a big fan of them :)

Reply:
Perhaps it makes sense to leave this snippet as a reference. These are guidelines after all not strict rules. IMHO I don't see downside of trailing commas - some noise at most. :)